app/models/teacher.rb

```diff
@@ -97,8 +90,7 @@ def self.user_from_omniauth(auth)
         user.email = auth.info.email
         user.admin = true
         user.status = 'Other'
-        user.validated = true
-        user.denied = false
+        user.application_status = 'Validated'
```

Comment:
Don't need this `if` after my PR gets merged. Consider merge that PR first then merge this branch with master

Reply:
Okay, I will wait and hopefully we can get that done by tomorrow